app/views/users/show.html.erb

```diff
@@ -1,11 +1,16 @@
-<%= stylesheet_link_tag "views" %>
+<%= stylesheet_link_tag "user_info" %>
 <!DOCTYPE html>
 <html>
 <head>
 	<meta charset="utf-8">
 	<title>specific user</title>
 </head>
 <body>
-  <h1>Here are information about specific user</h1>
+<main id ="container"> 
+  <%= render @user%>
+  <%= render partial: "bio" , locals:{bio:@user.bio}%>
+  <%= render partial: "user_information", as: "post" , collection: @user.recent_posts%>
+  <button id="show-posts">See all posts</button>
```

Comment:
`See All Posts` should be a link and opens the user's post index when the user clicks on it.